searx/engines/startpage.py

```diff
@@ -442,7 +442,12 @@ def fetch_traits(engine_traits: EngineTraits):
     for lang_code in filter(
         lambda lang_code: lang_code.find('_') == -1, babel.localedata.locale_identifiers()  # type: ignore
     ):
-        native_name = babel.Locale(lang_code).get_language_name().lower()  # type: ignore
+        try:
+            native_name = babel.Locale(lang_code).get_language_name().lower()  # type: ignore
+        except (TypeError, AttributeError) as e:
+            print(f"ERROR: {lang_code} is unknown by babel {e}")
+            continue
```

Comment:
I thin @Bnyro  is right / its in the fetch function and not related to this PR .. @allendema can you please split this hunk off into a separate PR / thanks :+1:

Reply:
Had to test my changes to ddg, startpage and qwant but `update_engine_traits.py` would fail without this patch. Supposedly the github checks would fail then too. Thats why I did not create a seperate pull request.

The fix for the babel error is indeed not mentioned in the commit but the pull request itself.

Reply:
I had to fix this issue in a dedicated PR / for the reasons see:

- https://github.com/searxng/searxng/pull/3525

I merged this PR #3525 into master and after, I rebased you branch on master.